#include <asyncio/uv.h>

DEFINE_ERROR_CATEGORY_INSTANCE(asyncio::uv::Error)
